### DISCUSSION TITLE
How to set default values for Templates

### DISCUSSION THREAD

**Discussion by raguila8** [tinacms/tinacms#3278] [General]
Hello, in the latest update, default values have a [new behavior](https://tina.io/blog/tina-version-0.69.13). The default values are now set at the collection level. How would you set default values for templates? Based on the current documentation it is not obvious how to do it. This is a basic version of what I have:

```javascript
{
      label: "Pages",
      name: "page",
      path: "content/pages",
      format: "md",
      defaultItem: () => {
        return {
          blocks: {
            contentBlock: {
              header: {
                title: "Our values",
                text: "Lorem ipsum dolor sit amet maecenas justo euismod justo erat a."             
              },
            },
          },
        };
      },
      fields: [
        {
          type: "object",
          list: true,
          name: "blocks",
          label: "Sections",
          ui: {
            visualSelector: true,
          },
          templates: [
            ContentBlockSchema,
           ],
        },
      ],
    },
```
where `pages` is a collection, and `ContentBlockSchema` is a template named `contentBlock`. This doesn't work. What is the new way of doing this? Thanks!

**Comment by raguila8** [comment]
Hello, disregard my previous comment. I was testing it with an already instantiated block with empty fields and expecting them to have the default values. It should work with newly created blocks.

**Reply by jamespohalloran** [reply]
That's right! We've simplified defaults so that they only apply to NEW content (E.g defaults for creating a new document, or a new list-item). They aren't injected at runtime for missing values.
